extensions/widgets/trending/src/components/cards/latest-topics.tsx

```diff
@@ -1,17 +1,26 @@
 import React, { useEffect, useRef, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import isEqual from 'lodash/isEqual';
+import difference from 'lodash/difference';
+import pullAll from 'lodash/pullAll';
 import getSDK from '@akashaorg/awf-sdk';
+import { useRootComponentProps } from '@akashaorg/ui-awf-hooks';
 import {
   useCreateInterestsMutation,
   useUpdateInterestsMutation,
 } from '@akashaorg/ui-awf-hooks/lib/generated/apollo';
+import { NotificationEvents, NotificationTypes } from '@akashaorg/typings/lib/ui';
 import Card from '@akashaorg/design-system-core/lib/components/Card';
 import Stack from '@akashaorg/design-system-core/lib/components/Stack';
 import Text from '@akashaorg/design-system-core/lib/components/Text';
 import TrendingWidgetLoadingCard from '@akashaorg/design-system-components/lib/components/TrendingWidgetLoadingCard';
 import { TopicRow } from './topic-row';
-import isEqual from 'lodash/isEqual';
-import difference from 'lodash/difference';
-import pullAll from 'lodash/pullAll';
+
+declare global {
+  interface Set<T> {
+    symmetricDifference: (value: Set<T>) => Set<T>;
+  }
+}
```

Comment:
@kenshyx is it okay, if we have this global defined here?

Reply:
why would you do something like this? this is a type that should come from the typescript lib

Reply:
merge with next, I've updated the tsconfig for you;
please remove this global overwrite